src/layout/index.tsx

```diff
@@ -4,17 +4,29 @@ import Sidebar from './Sidebar'
 import { Wrapper, MainContent, AppContent } from './styled'
 import Footer from './Footer'
 import URLWarning from 'src/components/Header/URLWarning'
+import MobileMenu from './Header/MobileMenu'
 
 const Layout: React.FC<unknown> = ({ children }) => {
   const [isDrawerCollapsed, setIsDrawerCollapsed] = useState(true)
 
+  const [activedMobileMenu, setActivedMobileMenu] = useState(false)

```

Comment:
there is typo here 
`activedMobileMenu` should be `activeMobileMenu`
`setActivedMobileMenu` should be `setActiveMobileMenu`